src/modules/esl-media/providers/html5/video-provider.ts

```diff
@@ -12,6 +15,13 @@ export class VideoProvider extends HTMLMediaProvider {
 
   protected override _el: HTMLVideoElement;
 
+  public constructor(component: ESLMedia, config: MediaProviderConfig) {
+    super(component, config);
+    window.removeEventListener('resize', this.refreshElementSize);
+  }
+
+  protected override onResize(): void {}
```

Comment:
Doesn't look right
Providers should not subsctbe/unsubscribe or change anything outside from constructor

And I'm sure we can make resize subscription management more predictable then such side effect